app/constants/versionCheck.js

```diff
@@ -0,0 +1,6 @@
+export const APPSTORE_URL = 'itms-apps://itunes.apple.com/us/app/id1508266966';
+export const PLAYSTORE_URL = 'market://details?id=org.pathcheck.covidsafepaths';
+export const VERSION_URL =
+  'https://rawcdn.githack.com/tripleblindmarket/covid-safe-paths/0008e269c355e14bb216bb6e69cf4b89e25012b2/versions.yaml';
```

Comment:
Please change to `https://rawcdn.githack.com/tripleblindmarket/covid-safe-paths/master/versions.yaml`.  This will still use the CDN with longer caching, but won't include a tag or commit hash